client/me/account-close/main.jsx

```diff
@@ -192,7 +192,7 @@ class AccountSettingsClose extends Component {
 						{ hasPurchases &&
 							! hasAtomicSites && (
 								<Button primary href="/me/purchases">
-									{ translate( 'Manage purchases' ) }
+									{ translate( 'Manage purchases', { context: 'button label' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Manage Purchases', { context: 'button label'} )` *ES Score: 20*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/close-account-button-context)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `button label`

 Would it make sense to reuse one of the above?